src/workforce-management/Views/TrainingProgram/Add.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@model workforce_management.ViewModels.TrainingProgramAdd
+
+<h2>Add a New Training Program</h2>
+
+<form asp-action="Add" method="post">
+    <div class="form-horizontal">
+        <hr />
+        <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+        <div class="form-group">
+            <label asp-for="NewTrainingProgram.Name" class="col-md-2 control-label"></label>
+            <div class="col-md-10">
+                <input asp-for="NewTrainingProgram.Name" class="form-control" />
+                <span asp-validation-for="NewTrainingProgram.Name" class="text-danger"></span>
+            </div>
+        </div>
+        <div class="form-group">
+            <label asp-for="NewTrainingProgram.Description" class="col-md-2 control-label"></label>
+            <div class="col-md-10">
+                <input asp-for="NewTrainingProgram.Description" class="form-control" />
+                <span asp-validation-for="NewTrainingProgram.Description" class="text-danger"></span>
+            </div>
+        </div>
+        <div class="form-group">
+            <label asp-for="NewTrainingProgram.Attendees" class="col-md-2 control-label">Add Employee</label>
+            <div class="col-md-10">
+                @Html.DropDownList("EmployeeIds", @Model.Employees, "Asign Employees", new { multiple = "true" })
```

Comment:
We should use a ListBox for this instead. Will be more consistent with the Edit page once someone gets into that.